learning_resources/views.py

```diff
@@ -293,3 +298,20 @@ def perform_destroy(self, instance):
         #     upsert_staff_list(staff_list.id)
         # else:
         #     deindex_staff_list(staff_list)
+
+
+class TopicViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for topics
+    """
+
+    class LargePagination(DefaultPagination):
```

Comment:
Maybe put this at the same level as `DefaultPagination` above so it can be used in the future by other viewsets where needed.